Samples/Samples/ViewModel/CalendarViewModel.cs

```diff
@@ -11,28 +13,77 @@ public class CalendarViewModel : BaseViewModel
         public CalendarViewModel()
         {
             GetCalendars = new Command(OnClickGetCalendars);
+            GetAllEvents = new Command(OnClickGetEvents);
             RequestCalendarReadAccess = new Command(OnRequestCalendarReadAccess);
             RequestCalendarWriteAccess = new Command(OnRequestCalendarWriteAccess);
         }
 
         public ICommand GetCalendars { get; }
 
+        public ICommand GetAllEvents { get; }
+
         public ICommand RequestCalendarReadAccess { get; }
 
         public ICommand RequestCalendarWriteAccess { get; }
 
         public ObservableCollection<ICalendar> Calendars { get; } = new ObservableCollection<ICalendar>();
 
+        public ObservableCollection<IEvent> Events { get; } = new ObservableCollection<IEvent>();
+
+        ICalendar selectedCalendar;
```

Comment:
Is this the correct ordering of public/private members?

Reply:
I'm gonna say no, correcting now